services/datamanager/builtin/builtin_test.go

```diff
@@ -848,20 +851,20 @@ func waitForCaptureFilesToEqualNFiles(ctx context.Context, captureDir string, n
 			for _, f := range files {
 				fNames = append(fNames, f.Name())
 			}
-			logger.Errorf("target: %d, iterations: %d, nonEmptyFiles: %d, files: %v", n, i, nonEmptyFiles, fNames)
+			logger.Errorf("target: %d, iterations: %d, nonEmptyFiles: %d, files: %v", n, i, captureFiles, fNames)
 			return err
 		}
 		files = getAllFileInfos(captureDir)
-		nonEmptyFiles = 0
+		captureFiles = 0
 		for idx := range files {
-			if files[idx].Size() > int64(emptyFileBytesSize) {
+			if files[idx].Size() > int64(emptyFileBytesSize) && filepath.Ext(files[idx].Name()) == data.CompletedCaptureFileExt {
```

Comment:
we should only be asesrting on the contents of capture files, not .prog files. .prog files, by definition, are still in the progress of being written to so their contents can change at any time, resulting in flakey tests.